packages/graphql-schema/tsconfig.json

```diff
@@ -2,15 +2,26 @@
   "extends": "@tsconfig/node18/tsconfig.json",
   "compilerOptions": {
     "target": "ES2020",
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "strict": true,
```

Comment:
👀

I dont mind any here and there but do we really wanna be removing this if the project is already compiling?

Reply:
They're already included verbatim in the `node18` config! I'm not removing any configuration here